lightningd/bitcoind.h

```diff
@@ -61,6 +61,8 @@ struct bitcoind *new_bitcoind(const tal_t *ctx,
 			      struct lightningd *ld,
 			      struct log *log);
 
+void report_bitcoind_unavailability_and_exit(struct bitcoind *bitcoind, const char *error_message);
```

Comment:
Why is this being exported? The function is afaik only being used from `bitcoin.c`.